Resources/views/CRUD/base_edit_form_macro.html.twig

```diff
@@ -1,10 +1,8 @@
-{% macro render_groups(admin, form, groups, has_tab) %}
-    <div class="row">
-
+{% macro render_groups(admin, form, groups, has_tab, no_padding = false) %}
+    {% if has_tab %}<div class="row">{% endif %}
     {% for code in groups %}
         {% set form_group = admin.formgroups[code] %}
-
-        <div class="{{ form_group.class | default('col-md-12') }}">
+        <div class="{{ form_group.class }} {{ no_padding ? "nopadding" }}"> {# default class 'col-md-12' removed because it do not work with tabs#}
```

Comment:
And not in the code IMO, that's what I implied (sorry for not saying it out loud) :P The word "removed" is about code change, and git is a better fit than code to store code history.